presto-main/src/main/java/com/facebook/presto/server/security/SecurityConfig.java

```diff
@@ -32,6 +32,7 @@
     private static final Splitter SPLITTER = Splitter.on(',').trimResults().omitEmptyStrings();
 
     private List<AuthenticationType> authenticationTypes = ImmutableList.of();
+    private Boolean authorizedIdentitySelectionEnabled = false;
```

Comment:
use boolean instead of Boolean ? No need to set False.

Reply:
Done.